### PR TITLE
Feature/glide 77

### PR DESCRIPTION
**Pull request by Horcon**
I'm currently starting the pull request to create a test environment for the frontend to use and confirm that the functionality is as desired.

### CODE REVIEW

**Comment by Horcon** [comment]
I feel confident that this PR is now ready to be reviewed. I haven't done much on the testing side of things, I'll look into how much that needs to be done once this PR is merged given just how much is to be reviewed at this point.

**Comment by Horcon** [comment]
As for the Code Climate issues, the 2 code duplications are a joke, and the underscores are from what I was provided by the frontend, so if they change the format I can/will change it, otherwise it has to stay to process the JSON.

File: models/TrackPoint.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+type TrackPoint struct {
+	Time         int64
+	Latitude     float64
+	Longitude    float64
+	Valid        bool
+	Pressure_alt int
+	GPS_alt      int
+	Accuracy     float64
+	Engine_RPM   float64
+}
```
Comment (MarkusAJacobsen) [inline models/TrackPoint.go:13]: type TrackPoint struct {
	Time         int64 `json:"Type"`
	Latitude     float64 `json:"Latitude"`
	Longitude    float64 `json:"Longitude"`
	Valid        bool `json:"Valid"`
	PressureAlt int `json:"Pressure_alt"`
	GPSAlt      int `json:"GPS_alt"`
	Accuracy     float64 `json:"Accuracy"`
	EngineRPM   float64 `json:"Engine_RPM"`
}

Comment (MarkusAJacobsen) [reply models/TrackPoint.go:13]: json is enclosed by `  but git is interpreting them as a code snippet

File: rest/dbHandler.go [hunk 2]
```diff
@@ -138,7 +118,73 @@ func (dbHandler DbHandler) deleteTrackPage(w http.ResponseWriter, r *http.Reques
...
+	// Parse into objects the string JSON from header
+	var parsed []models.TrackPoint
+	err := json.Unmarshal([]byte(rawJSON), &parsed)
+	if err != nil {
+		gtbackend.DebugLogErrNoMsg(log, err)
+
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
```
Comment (MarkusAJacobsen) [inline rest/dbHandler.go:173]: if err := json.Unmarshal([]byte(rawJSON), &parsed); err != nil {
    .......
}

**Review by MarkusAJacobsen — CHANGES_REQUESTED** [review]

File: models/TrackPoint.go (hunk 1 above)
Comment (Horcon) [reply models/TrackPoint.go:13]: Interesting, done.

File: rest/dbHandler.go (hunk 2 above)
Comment (Horcon) [reply rest/dbHandler.go:173]: Guess it cuts down on a line.

Comment (MarkusAJacobsen) [reply rest/dbHandler.go:173]: Its a syntax shortcut which should be used imo

Comment (MarkusAJacobsen) [reply rest/dbHandler.go:173]: It's a good to know, approving

**Review by MarkusAJacobsen — DISMISSED** [review]

**Review by MarkusAJacobsen — APPROVED** [review]

**Pull request merged**